### PR TITLE
[NUI] Add tests for some WebView APIs.

### PR DESCRIPTION
**Pull request by huayongxu**
### Description of Change ###

This is a basic patch for WebView APIs verification.

### API Changes ###

 - ACR:

### CODE REVIEW

File: test/Tizen.NUI.WebViewTest/SimpleWebViewApp.cs [hunk 1]
```diff
@@ -0,0 +1,831 @@
...
+                {
+                    Log.Info("WebView", $"key XF86Green is pressed.");
+
+                    //greenKeyPressedCount++;
+                    if (greenKeyPressedCount % 2 == 0)
+                    {
+                        // webview apis
+                        Log.Info("WebView", $"web page title is {simpleWebView.Title}");
+
+                        //faviconView = simpleWebView.Favicon;
```
Comment (zg2nets) [inline test/Tizen.NUI.WebViewTest/SimpleWebViewApp.cs:502]: plz remove useless code

File: test/Tizen.NUI.WebViewTest/SimpleWebViewApp.cs [hunk 2]
```diff
@@ -0,0 +1,831 @@
...
+                        simpleWebView.CanGoBack();
+                        simpleWebView.CanGoForward();
+                        simpleWebView.CheckVideoPlayingAsynchronously(OnVideoPlaying);
+                        simpleWebView.ClearAllTilesResources();
+                        simpleWebView.ClearHistory();
+                        simpleWebView.EvaluateJavaScript("document.body.style.backgroundColor='yellow';");
+                        Log.Info("WebView", $"web view, ScaleFactor is {simpleWebView.GetScaleFactor()}");
+
+                        Rectangle viewArea = new Rectangle(0, 0, 20, 20);
+                        //ImageView shotView = simpleWebView.GetScreenshot(viewArea, 1.0f);
```
Comment (zg2nets) [inline test/Tizen.NUI.WebViewTest/SimpleWebViewApp.cs:540]: plz remove useless code

File: test/Tizen.NUI.WebViewTest/SimpleWebViewApp.cs [hunk 3]
```diff
@@ -0,0 +1,831 @@
...
+                        simpleWebView.LoadHtmlString("<Html><Head><title>Example</title></Head><Body><b>[This text is Bold......]</b></Body></Html>");
+                        simpleWebView.ScrollPosition = new Position(0, 200);
+                        simpleWebView.ScrollBy(0, 50);
+                        simpleWebView.ScrollEdgeBy(0, 50);
+
+                        Log.Info("WebView", $"scroll position is ({simpleWebView.ScrollPosition.X}, {simpleWebView.ScrollPosition.Y}).");
+                        Log.Info("WebView", $"scroll size is ({simpleWebView.ScrollSize.Width}, {simpleWebView.ScrollSize.Height}).");
+                        Log.Info("WebView", $"content size is ({simpleWebView.ContentSize.Width}, {simpleWebView.ContentSize.Height}).");
+
+                        //simpleWebView.StartInspectorServer(8080);
```
Comment (zg2nets) [inline test/Tizen.NUI.WebViewTest/SimpleWebViewApp.cs:611]: plz remove useless code

File: test/Tizen.NUI.WebViewTest/SimpleWebViewApp.cs [hunk 4]
```diff
@@ -0,0 +1,831 @@
...
+                        simpleWebView.RemoveCustomHeader("test");
+                        Log.Info("WebView", $"web view, here");
+                        simpleWebView.Suspend();
+                        simpleWebView.SuspendNetworkLoading();
+                        Log.Info("WebView", $"web view, here");
+                        simpleWebView.Resume();
+                        simpleWebView.ResumeNetworkLoading();
+                        Log.Info("WebView", $"web view, here");
+
+                        //simpleWebView.LoadContents();
```
Comment (zg2nets) [inline test/Tizen.NUI.WebViewTest/SimpleWebViewApp.cs:600]: plz remove useless code

File: test/Tizen.NUI.WebViewTest/SimpleWebViewApp.cs [hunk 5]
```diff
@@ -0,0 +1,831 @@
...
+            Log.Info("WebView", $"------------response policy decided, PolicyDecisionType: {e.ResponsePolicyDecisionMaker.PolicyDecisionType}-------");
+            Log.Info("WebView", $"------------response policy decided, ResponseMime: {e.ResponsePolicyDecisionMaker.ResponseMime}-------");
+            Log.Info("WebView", $"------------response policy decided, ResponseStatusCode: {e.ResponsePolicyDecisionMaker.ResponseStatusCode}-------");
+            Log.Info("WebView", $"------------response policy decided, DecisionNavigationType: {e.ResponsePolicyDecisionMaker.DecisionNavigationType}-------");
+            Log.Info("WebView", $"------------response policy decided, Scheme: {e.ResponsePolicyDecisionMaker.Scheme}-------");
+            if (e.ResponsePolicyDecisionMaker.Frame != null)
+            {
+                Log.Info("WebView", $"------------response policy decided, Frame.IsMainFrame: {e.ResponsePolicyDecisionMaker.Frame.IsMainFrame}-------");
+            }
+            //e.ResponsePolicyDecisionMaker.Ignore();
```
Comment (zg2nets) [inline test/Tizen.NUI.WebViewTest/SimpleWebViewApp.cs:305]: plz remove useless code

**Review by dongsug-song — APPROVED** [review]
looks good to me

**Pull request merged**